### DISCUSSION TITLE
Why does the Coqui.ai website only need 3 seconds?

### DISCUSSION THREAD

**Discussion by Hexatona** [coqui-ai/TTS#2554] [General Q&A] [answered]
Hi!  I'm sorry if this has been asked before, but I looked around and didn't see any simmilar questions.  Basically, I noticed the Coqui.ai website claiming it can clone voices with 3 seconds//1 minute  - any really limited number of audio, but most of the guides I've seen on training your own voice manually usually require absolute gobs of audio - ideally like 8 hours.  Can anyone explain why there's such a huge disparity in the audio required?

**Comment by Harrolee** [comment, marked as answer]
Hi @Hexatona! Checkout the [YourTTS repo](https://github.com/Edresson/YourTTS).
YourTTS is the original work of the Coqui team. This is the secret sauce seen on the Coqui.ai website. [Here's a notebook](https://colab.research.google.com/drive/1WArisOG8vLGvrnoaLyEBOlJ0jG3LDtc2?usp=sharing) for the "3 seconds//1 minute" audio voice clone experience.

Here's a related [gh discussion post](https://github.com/coqui-ai/TTS/discussions/653)
